automation_tools/__init__.py

```diff
@@ -1032,7 +1027,9 @@ def setup_foreman_discovery(sat_version):
         run('wget -nv -O- {0} | tar x --overwrite -C /var/lib/tftpboot/boot'.format(image_url))
     else:
         # Since 6.3, installer should install all required packages except FDI
-        run('rpm -q {0}'.format(' '.join(packages)))
+        if sat_version in ('6.5', '6.6'):
```

Comment:
We don't need to unlock packages for 6.5.

Reply:
I checked [6.3, 6.4, 6.5, 6.6] simply by running ```foreman-maintain packages unlock```
and only on [6.5, 6.6] the RC = 0 

But yeah the truth is that on 6.5 it actually does nothing. Going to amend then :)